tests/css_quotes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -288,7 +288,7 @@ one 'two' three {}
 @charset 'UTF-8'; /* Single quotes are invalid here. Keep them since we don't know what the user is doing. */
 
 @supports (content: one 'two' three 'four') {
-  a[href='foo' y],
+  a[href='foo'],
```

Comment:
@chriseppstein says what is 4/5 version postcss selector parser support custom modificator

Reply:
We support nonstandard modifier. the value is stored in `raws.insensitiveFlag`

https://github.com/postcss/postcss-selector-parser/blob/master/src/__tests__/attributes.js#L396-L402

I think the API could be better around this, so I filed an [issue to track it](https://github.com/postcss/postcss-selector-parser/issues/135) a while ago.

Reply:
@chriseppstein thanks for clarify. `raws.insensitiveFlag` looks some mesleading name for non standard flags

Reply:
I agree that's why I filed the bug.